source/_components/device_tracker.mqtt.markdown

```diff
@@ -26,15 +26,45 @@ device_tracker:
 
 {% configuration %}
 devices:
-  description: List of devices with their topic.
+  description: A dictionary where the keys are devices and the values are their corresponding topic.
   required: true
-  type: list
+  type: map
 qos:
   description: The QoS level of the topic.
   required: false
   type: integer
+payload_home:
+  description: The payload that represents the home state for the device.
+  required: false
+  type: string
+  default: `home`
+payload_not_home:
+  description: The payload that represents the not home state for the device.
+  required: false
+  type: string
+  default: `not_home`
+payload_custom:
+  description: A dictionary to map payload values to custom zones (e.g. not `home` and `not_home`). The keys are the payloads and the values are the corresponding locations that the payload represents.
+  required: false
+  type: string
```

Comment:
and maybe this one also

Reply:
see my other comment

Reply:
oops I committed a change because I thought I made a mistake, did not see that you had changed this. Let me know if you would like me to change it back, but both this and `devices` look like maps to me